backend/de.metas.banking/de.metas.banking.base/src/main/java/de/metas/banking/payment/impl/PaySelectionUpdater.java

```diff
@@ -396,7 +396,7 @@ else if (PaySelectionMatchingMode.CREDIT_TRANSFER_TO_CUSTOMER.equals(matchRequir
 				+ " o.Bill_BPartner_ID as C_BPartner_ID," // 8
 				// C_BP_BankAccount_ID
 				+ " (SELECT max(bpb.C_BP_BankAccount_ID) FROM C_BP_BankAccount bpb WHERE bpb.C_BPartner_ID = o.Bill_BPartner_ID AND bpb.IsActive='Y' "
-				+ " AND bpb.IBAN IS NOT NULL AND bpb.C_Currency_ID = o.C_Currency_ID) as C_BP_BankAccount_ID "  //9
+				+ " AND (bpb.IBAN IS NOT NULL OR bpb.QR_IBAN IS NOT NULL) AND bpb.C_Currency_ID = o.C_Currency_ID) as C_BP_BankAccount_ID "  //9
```

Comment:
what's the reason for this where clause?
what's the impact? (how behaved before, how behaves after)